### PR TITLE
specvol() 75-term

### PR DESCRIPTION
**Pull request by castelao**
specvol() based on 75-term equation

doctest values obtained from GSW-Matlab 3.0.5

Temporarily disabling test_profile on specvol. This is not the best way to do it, but  test_profile was meant for 48-term and looks like it wasn't validated for Matlab's specvol anyways. Therefore, approval of test_profile() doesn't guarantee it is correct anyways.

Once there are more converted functions, it would be a good idea to update and reactivate test_profile for specvol().

There are more functions ready to PR. I'm submitting this first as a test.

### CODE REVIEW

**Comment by efiring** [comment]
@ocefpaf I guess I'm 80% committed to an eventual hybrid in which at least the core functions with the polynomials are wrapped C, but the functions like Nsquared and the geostrophic calculations are reimplemented in Python.  I think this gives us the best of both worlds.  There is a major performance and memory advantage in using C for the the big polynomials, in addition to the advantage of reducing the number of different codebases in which they are defined.

File: gsw/gibbs/density_enthalpy_75.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+    Notes
+    -----
+    Note that this 75-term equation has been fitted in a restricted range of
+    parameter space, and is most accurate inside the "oceanographic funnel"
+    described in McDougall et al. (2003).  The GSW library function
+    "infunnel(SA,CT,p)" is avaialble to be used if one wants to test if some of
+    one's data lies outside this "funnel".
+
+    SA & CT need to have the same dimensions.
```
Comment (efiring) [inline gsw/gibbs/density_enthalpy_75.py]: It's not critical initially, but longer term we need to make the documentation a better match for the code: what we are actually doing is just letting the numpy broadcasting rules apply.  Therefore the statements about dimensions here are not quite right.  I would just delete everything about dimensions.

Comment (castelao) [inline gsw/gibbs/density_enthalpy_75.py]: Done

**Comment by efiring** [comment]
It looks like the git commit history is a bit confusing, with all the merges.  Maybe this can all be fixed up via a "rebase -i" on current master?

**Comment by ocefpaf** [comment]
@efiring I believe you can squash and merge with GitHub interface now if that is an acceptable solution for simplifying the commit history here.

**Pull request merged**